verification/infrastructure/models.py

```diff
@@ -0,0 +1,34 @@
+from sqlalchemy import Column, Integer
+from sqlalchemy.dialects.mysql import JSON, TIMESTAMP, VARCHAR
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class VerificationModel(Base):
+    __tablename__ = "verification"
+    id = Column("id", VARCHAR(225), primary_key=True)
+    verification_type = Column("verification_type", VARCHAR(225), primary_key=True)
+    entity_id = Column("entity_id", VARCHAR(255))
+    status = Column("status", VARCHAR(255))
+    requestee = Column("requestee", VARCHAR(255))
+    created_at = Column("created_at", TIMESTAMP(timezone=False))
+    updated_at = Column("updated_at", TIMESTAMP(timezone=False))
```

Comment:
Add column verified_at

Reply:
verified state is last state change so we don't need separate column for that